frontend/src/models/cohortsModel.ts

```diff
@@ -58,6 +58,13 @@ export const cohortsModel = kea<cohortsModelType>({
 
     selectors: {
         cohortsWithAllUsers: [(s) => [s.cohorts], (cohorts) => [{ id: 'all', name: 'All Users*' }, ...cohorts]],
+        cohortsIdMapped: [
```

Comment:
Nit, feel free to ignore, but I think we should describe an object as what it is, not how it came to be. I've used the `ByX` notation in the past, which would be `cohortsById` in this case. `Map` is a separate object in JS as well, making it just a bit more confusing. Might be just me though, so your mileage may vary.

Reply:
:+1: `ByX` is better